ci/build_setup.sh

```diff
@@ -26,23 +34,23 @@ function setup_gcc_toolchain() {
     export BAZEL_COMPILER=gcc
     echo "$CC/$CXX toolchain configured"
   else
-    export BAZEL_BUILD_OPTIONS="--config=remote-gcc ${BAZEL_BUILD_OPTIONS}"
+    export BAZEL_BUILD_OPTIONS=("--config=remote-gcc" "${BAZEL_BUILD_OPTIONS[@]}")
   fi
 }
 
 function setup_clang_toolchain() {
   ENVOY_STDLIB="${ENVOY_STDLIB:-libc++}"
   if [[ -z "${ENVOY_RBE}" ]]; then
     if [[ "${ENVOY_STDLIB}" == "libc++" ]]; then
-      export BAZEL_BUILD_OPTIONS="--config=libc++ ${BAZEL_BUILD_OPTIONS}"
+      export BAZEL_BUILD_OPTIONS=("--config=libc++" "${BAZEL_BUILD_OPTIONS[@]}")
```

Comment:
So I gather these `export` keywords don't have the expected effect because `BAZEL_BUILD_OPTIONS` is an array. Is there a reason to keep the export keywords (here and in a bunch of other places)? I know they confused me into not understanding why the clang settings weren't applied in `run_clang_tidy.sh`.

Reply:
yep, no - you are right - arrays arent exported so better remove - ill update